config/config/wc-config.yaml

```diff
@@ -269,9 +269,18 @@ networkPolicies:
       # ip of all nodes in the cluster for internal communication
       ips:
         - set-me
-  certManager:
-    enabled: true
   monitoring:
     enabled: true
   fluentd:
     enabled: true
+
+  ingressNginx:
+    enabled: true
+    ingressOverride:
+      enabled : false
+      ips:
+      - set-me-if-enabled
+
+  certManager:
+    namespaces:
+      - set-me
```

Comment:
Since user namespaces automatically are included, couldn't this be by default `[]`?

Reply:
I agree.

Reply:
I will change it.